recipes/sasktran2/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "sasktran2" %}
+{% set version = "2023.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usask-arg/sasktran2/archive/v{{ version }}.tar.gz
+  sha256: 073d274f69e03e6f04297e08c496742ff408da5ab9bc6cf75d4a74bb634988fa
+
+build:
+  binary_relocation: true
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv 
+  number: 0
+  skip: true  # [py<310]
+
+requirements:
+  build:
+    - cmake >=3.5
+    - make  # [not win]
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}  # [not win]
+  host:
+    - python {{ python }}
+    - pip
+    - eigen
+    - libblas =*=*netlib
+    - libcblas =*=*netlib
+    - liblapack =*=*netlib
+    - liblapacke =*=*netlib
+    - libtmglib =*=*netlib
```

The global pinning will take care of that.
```suggestion
    - libblas
    - libcblas
    - liblapack
    - liblapacke
    - libtmglib
```